src/mongo/dbtests/dbtests.cpp

```diff
@@ -156,6 +158,11 @@ int dbtestsMain(int argc, char** argv, char** envp) {
         ->setFollowerMode(repl::MemberState::RS_PRIMARY)
         .ignore();
 
+    repl::DropPendingCollectionReaper::set(
+                service,
+                std::make_unique<repl::DropPendingCollectionReaper>(new repl::StorageInterfaceMock())
```

Comment:
Please use stdx::make_unique() instead of std::make_unique(). FYI, there's a built-in linter you can run using the build system:

scons.py .... lint